projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -1,26 +1,51 @@
 <div class="go-table-container" *ngIf="showTable">
-  <table class="go-table" *ngIf="hasData(); else noDataTable">
-    <thead>
-      <tr>
-        <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
-          <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
-          <ng-template #columnHeaderOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
-          </ng-template>
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr *ngFor="let item of localTableConfig.tableData" class="go-table__tr">
-        <td *ngFor="let col of columns" class="go-table__td" width="{{ col.width || 'auto' }}">
-          <ng-container *ngIf="!col.goTableCell; else columnCellOutlet">{{ col.getFieldData(item) }}</ng-container>
-          <ng-template #columnCellOutlet>
-            <ng-container *ngTemplateOutlet="col.goTableCell; context: { $implicit: item }"></ng-container>
-          </ng-template>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+  <div class="go-table__table">
+    <table class="go-table" *ngIf="hasData(); else noDataTable">
+      <thead>
+        <tr>
+          <th *ngFor="let col of columns" class="go-table__th" [ngClass]="columnClasses(col.field)" (click)="toggleSort(col.field)">
+            <ng-container *ngIf="!col.goTableHead; else columnHeaderOutlet">{{ col.title || col.field }}</ng-container>
+            <ng-template #columnHeaderOutlet>
+              <ng-container *ngTemplateOutlet="col.goTableHead"></ng-container>
+            </ng-template>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let item of setDisplayData()" class="go-table__tr">
+          <td *ngFor="let col of columns" class="go-table__td" width="{{ col.width || 'auto' }}">
```

Comment:
What if we called this class `go-table__column`?

Reply:
Thought about this, kind of along the same lines as my comment about the `heading`